From: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Subject: mm: include linux/pgtable.h in vma_internal.h
Date: Sat, 24 Aug 2024 20:20:17 +0100

For some arches this is required in order to have access to
USER_PGTABLES_CEILING and FIRST_USER_ADDRESS.

Link: https://lkml.kernel.org/r/da1a6b2c-f792-432f-b99f-51717dcc8d83@lucifer.local
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202408242348.uGvgH9tt-lkp@intel.com/
Closes: https://lore.kernel.org/oe-kbuild-all/202408242304.1A1fXTgE-lkp@intel.com/
Signed-off-by: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vma_internal.h |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/vma_internal.h~mm-vma-track-start-and-end-for-munmap-in-vma_munmap_struct-fix
+++ a/mm/vma_internal.h
@@ -33,6 +33,7 @@
 #include <linux/mutex.h>
 #include <linux/pagemap.h>
 #include <linux/pfn.h>
+#include <linux/pgtable.h>
 #include <linux/rcupdate.h>
 #include <linux/rmap.h>
 #include <linux/rwsem.h>
_
